webui/react/src/hooks/useAuthCheck.ts

```diff
@@ -17,7 +18,8 @@ const useAuthCheck = (): (() => void) => {
   useEffect(() => {
     const checkAuth = async (signal: AbortSignal): Promise<void> => {
       const authToken = globalStorage.authToken;
```

Comment:
suggestion: just this line should be enough? 
`const authToken = globalStorage.authToken || getCookie(AUTH_COOKIE_KEY)`

Reply:
moving cookie in front of globalStorage as a priority